doc/adr/0004-release-with-continuous-deployment.md

```diff
@@ -0,0 +1,66 @@
+# 4. Release with continuous deployment
+
+Date: 2023-06-22
+
+## Status
+
+TBC
+
+## Context
+
+The Accredited programmes service is currently about to launch a private beta,
+piloting with a small number of prisons, and as such we are focused on
+delivering quickly which requires as little release overhead as possible.
+
+MOJ provide the [tools for continuous
+deployment](https://user-guide.cloud-platform.service.justice.gov.uk/documentation/deploying-an-app/using-circleci-for-continuous-deployment.html),
+configured as part of the
+[TypeScript](https://github.com/ministryofjustice/hmpps-template-typescript/blob/eac50e97924df530e1649dc4ccd4339f4491789d/.circleci/config.yml#L148)
+and
+[Kotlin](https://github.com/ministryofjustice/hmpps-template-kotlin/blob/4c7bc649f8bf8deb8dc49af58c90fd1e9e085437/.circleci/config.yml#L66)
+template repositories. This is an approach taken by other projects at MOJ.
+
+As an external team we need to be mindful that when handing over there will be
+advantages to having a release process consistent with other MOJ services.
```

Comment:
This is interesting positionality! Does it need a bit more context if non-dxw devs are reading this two years from now? Might they read it as AcP permanently being managed by an external team?

Reply:
I've added (at time of writing) - we have a date at the top of this doc so hopefully that's clear enough.

Reply:
Sounds good! I guess I'd see the date as the date of adoption rather than necessarily the rough time in which this was applicable?